MediaGallery/MediaGallery/PickMedia.ios.cs

```diff
@@ -145,9 +147,9 @@ public override void DidFinishPicking(PHPickerViewController picker, PHPickerRes
 
             static IEnumerable<IMediaFile> ConvertPickerResults(PHPickerResult[] results)
                 => results
-                .Select(res => res.ItemProvider)
+                .Select(res => new { res.ItemProvider, res.AssetIdentifier })
```

Comment:
How will this be handled for older iOS versions?

Reply:
I commit an update to solve the anonymous type.